src/languages/lsl.js

```diff
@@ -0,0 +1,90 @@
+/*
+Language: Linden Scripting Language
+Description: The Linden Scripting Language is used in Second Life by Linden Labs.
+Author: Builder's Brewery <buildersbrewery@gmail.com>
+Category: scripting
+*/
+
+function(hljs) {
+
+    var LSL_STRING_ESCAPE_CHARS = {
+        className: 'subst',
+        begin: '\\[\\\\nt"\\]'
```

Comment:
This looks like it has too many back-slashes in it. In this form it will only match a single literal string `[\nt"]` which is probably not what it's supposed to mean? If LSL strings are simply backslah-escaped then we have a standard mode for that: `hljs.BACKSLASH_ESCAPE`.